lib/ansible/plugins/inventory/scaleway.py

```diff
@@ -0,0 +1,136 @@
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: scaleway
+    plugin_type: inventory
+    authors:
+      - Remy Leone <rleone@online.net>
+    short_description: Scaleway inventory source
+    description:
+        - Get inventory hosts from Scaleway
```

Comment:
you didn't document the 'regions' or 'tags' options

Reply:
done. Do you think it's clear enough?

Reply:
@bcoca Do you think the documentation is clear enough?

Reply:
Sorry you fixed the review. All good :)